ofl/bevietnampro/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,13 @@
+<p>
+  Be Vietnam Pro inherits the spirit of the
+  <a href="https://github.com/bettergui/BeVietnam">Be Vietnam</a> and takes it
+  to the next level. We refined Vietnamese letterforms with diacritics adaptive
+  forms for specific use cases and engineered them for better readability.
```

Comment:
Knowing that Be Vietnam is going to be deprecated and hidden from users, I would suggest to not add link and to clarify the description as if Be Vietnam didn't exist